py3status/modules/volume_status.py

```diff
@@ -23,10 +23,12 @@
         (default None)
     device: Device to use. Defaults value is backend dependent
         (default None)
+    device_is_speaker: Should this control the speakers (True) or the microphone (False)?
+        (default True)
     format: Format of the output.
-        (default '♪: {percentage}%')
+        (default '[\?if=device_is_speaker ♪|😮]: {percentage}%')
```

Comment:
@tobes `format = '[\?if=is_microphone 😮|♪]: {percentage}%'` ?

I think having `is_microphone = True` is nicer than `device_is_speaker = False` in the config.